vision/landmark-detection/pom.xml

```diff
@@ -44,6 +44,12 @@
       <groupId>com.google.api-client</groupId>
       <artifactId>google-api-client</artifactId>
       <version>1.22.0</version>
+      <exclusions>
+        <exclusion> <!-- exclude an old version of Guava -->
```

Comment:
Should we ask the Veneer folks to do this for us?   It doesn't strike me as something we should be promoting as the way to use our libraries in our samples.